frontend/src/scenes/organization/Invites/CreateOrgInviteModal.tsx

```diff
@@ -47,7 +47,7 @@ export function CreateOrgInviteModalWithButton({ type = 'button' }: { type?: 'bu
                         setIsVisible(true)
                     }}
                 >
-                    Invite Team Member
+                    Invite Teammate
```

Comment:
Note: Ideally this rename could have been split out into a separate short-lived PR - since there's 21 days worth of commits here, can we be sure no-one added any other `Team member` calls in a different PR.

Reply:
That's true